packages/genesys-spark-components/src/test/expectToHaveNoViolations.ts

```diff
@@ -0,0 +1,71 @@
+import { Result, NodeResult } from 'axe-core';
```

Comment:
This is the main change in this PR.
I added the ability to use the same axe exclusions array to the Playwright tests.